client/src/pages/CreateLesson/CreateLesson.js

```diff
@@ -9,6 +9,11 @@ import { createLesson } from 'api'
 import { useErrorWrap } from 'hooks'
 
 const CreateLesson = ({ navigation }) => {
+  // Closes the modal
```

Comment:
Any reason why the location of this method changed?

Reply:
To match the CreateUnit